unit_scaling/functional.py

```diff
@@ -186,6 +195,21 @@ def layer_norm(
     return F.layer_norm(input, normalized_shape, weight, bias, eps)
 
 
+@docstring_from(
+    torch.add,
+    short_description="Applies a **unit-scaled** addition.",
+    unsupported_args=["alpha"],
+)
+def add(
+    input: Tensor, other: Tensor, alpha: int = 1, out: Optional[Tensor] = None
```

Comment:
Worth adding a `constraint`? If so, would this be the first default=unconstrained op, though?